### PR TITLE
Set alt value automatically

### PR DESCRIPTION
**Pull request by rlho**
#9 
I tried the article you shared, but it didn't automatically generate the alt, so I created a new component for alt text.

### CODE REVIEW

File: schemas/Room.js [hunk 1]
```diff
@@ -1,6 +1,7 @@
 // ICONS
-import { MdYard } from "react-icons/md"
-import DimensionInput from '../src/DimensionInput'
+import { MdYard } from "react-icons/md";
```
Comment (rlho) [inline schemas/Room.js:2]: This is an auto-completion by prettier.

File: schemas/Room.js [hunk 2]
```diff
@@ -96,60 +99,74 @@ export default {
...
+            },
+          ],
+        },
+      ],
     },
     {
       title: "Background image",
       name: "backgroundImage",
       type: "image",
+      hotspot: true,
```
Comment (rlho) [inline schemas/Room.js:122]: This is the part I changed.

**Comment by hobg0blin** [comment]
So this is a good start, but what I'd like to get to is storing the "alt" attribute for an image so that if it's selected and already has an alt attribute, **that alt attribute is stored in the database so it doesn't have to be retyped**.

For example, if I select this door image:
image
I have to add the alt text every time, even though it's the same image (as far as I can tell, this is also true for this new feature). Ideally you should enter the alt text once, then have that field associated with the image permanently (but editable).

**Comment by hobg0blin** [comment]
For example, it looks like this media browser plugin might have the functionality we're looking for: https://www.sanity.io/plugins/sanity-plugin-media

**Pull request merged**